composer/algorithms/stochastic_depth/stochastic_depth.py

```diff
@@ -50,29 +56,32 @@ def apply_stochastic_depth(model: torch.nn.Module,
 
     .. note::
 
-        Stochastic Depth only works on instances of `torchvision.models.resnet.ResNet` for now.
+        Stochastic Depth only works on instances of :class:`torchvision.models.resnet.ResNet`
+        for now.
 
     Args:
-        model (torch.nn.Module): model containing modules to be replaced with stochastic versions
+        model (:class:`torch.nn.Module`): model containing modules to be replaced with 
+            stochastic versions.
         target_layer_name (str): Block to replace with a stochastic block
             equivalent. The name must be registered in ``STOCHASTIC_LAYER_MAPPING``
             dictionary with the target layer class and the stochastic layer class.
             Currently, only :class:`torchvision.models.resnet.Bottleneck` is supported.
-        stochastic_method (str, optional): The version of stochastic depth to use. ``"block"``
-            randomly drops blocks during training. ``"sample"`` randomly drops
-            samples within a block during training. Default: ``"block"``.
-        drop_rate (float, optional): The base probability of dropping a layer or sample. Must be
-            between 0.0 and 1.0. Default: `0.2``.
+        stochastic_method (str, optional): The version of stochastic depth to use. 
+            ``"block"`` randomly drops blocks during training. ``"sample"`` randomly
+            drops samples within a block during training. Default: ``"block"``.
+        drop_rate (float, optional): The base probability of dropping a layer or sample. 
+            Must be between 0.0 and 1.0. Default: `0.2``.
         drop_distribution (str, optional): How ``drop_rate`` is distributed across
             layers. Value must be one of ``"uniform"`` or ``"linear"``.
             ``"uniform"`` assigns the same ``drop_rate`` across all layers.
-            ``"linear"`` linearly increases the drop rate across layer depth
-            starting with 0 drop rate and ending with ``drop_rate``. Default: ``"linear"``.
-        use_same_gpu_seed (bool, optional): Set to ``True`` to have the same layers dropped
-            across GPUs when using multi-GPU training. Set to ``False`` to
-            have each GPU drop a different set of layers. Only used
+            ``"linear"`` linearly increases the drop rate across layer depth,
+            starting with 0 drop rate and ending with ``drop_rate``. 
+            Default: ``"linear"``.
+        use_same_gpu_seed (bool, optional): Set to ``True`` to have the 
+            same layers dropped across GPUs when using multi-GPU training. 
+            Set to ``False`` to have each GPU drop a different set of layers. Only used
             with ``"block"`` stochastic method. Default: ``True``.
-        optimizers (torch.optim.Optimizer | Sequence[torch.optim.Optimizer], optional):
+        optimizers (:class:`torch.optim.Optimizer` | Sequence[:class:`torch.optim.Optimizer`], optional):
```

Comment:
only catching this now, but these are already resolving correctly without the need for a `:class:` marker, see: https://docs.mosaicml.com/en/latest/api_reference/composer.algorithms.stochastic_depth.stochastic_depth.html#composer.algorithms.stochastic_depth.stochastic_depth.apply_stochastic_depth